.docs/user-guide/storage-providers.md

```diff
@@ -707,3 +707,107 @@ libstorage:
   in place, it may be possible for a client to attach a volume that is already
   attached to another node. Mounting and writing to such a volume could lead to
   data corruption.
+
+## GCEPD
+
+The Google Compute Engine (GCE) Persistent Disk driver registers a driver named
```

Comment:
Hi @codenrhoden,

It's up to you really. I only pointed it out because it may be the only place we reference GCE? I just want to avoid confusion if possible. If you don't think it's confusing then leave it. It just stood out, but I do not feel strongly about it.